thats_the_way.py

```diff
@@ -0,0 +1,16 @@
+import os
+
+def find_deep_files(path):
+    deep_files = []
+    for file in os.listdir(path):
+        if file.startswith("deep"):
+            deep_files.append(file)
+    return deep_files
+
+path = "C:\\Users\\yisra\\Downloads\\Notebooks-master\\week05\\images"
+
+if os.path.exists(path):
+    deep_files = find_deep_files(path)
+    print(deep_files)
+else:
+    print(f"The path {path} does not exist.")
```

Comment:
Separate code to small functions- 
https://softwareengineering.stackexchange.com/questions/308108/when-is-it-appropriate-to-make-a-separate-function-when-there-will-only-ever-be

Reply:
I fixed it